src/content/en/updates/2016/12/lighthouse-dbw.md

```diff
@@ -0,0 +1,87 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Lighthouse. Redesign, new best practice audits, and an online report viewer.
+
+{# wf_updated_on: 2016-12-14 #}
+{# wf_published_on: 2016-12-14 #}
+{# wf_tags: lighthouse,tools,performance,pwa #}
+{# wf_featured_image: /web/progressive-web-apps/images/pwa-lighthouse.png #}
+{# wf_featured_snippet: What's new in Lighthouse. Redesign, new best practice audits, and an online report viewer. #}
+
+<style>
+figure {
+  text-align: center;
+}
+.lighthouse-logo {
+  height: 150px;
+  width: auto;
+}
+</style>
+
+# Building a Better Web with Lighthouse {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+[Lighthouse](/web/tools/lighthouse/) is an [open-source](https://github.com/GoogleChrome/lighthouse), automated tool for improving the quality of your web apps. You can install it as a [Chrome Extension](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) or run it as a Node command line tool. When you give Lighthouse a URL, it runs a barrage of tests against the page and then generates a report explaining how well the page did and indicating areas for improvement.
```

Comment:
I've been enforcing this for the last few months and consider it a requirement. Please break lines at 80 chars.